main.py

```diff
@@ -318,25 +322,15 @@ def get_batch_bg(args, item, channel, intervals):
             
         # Expand each interval, combine with scores, and contract to smaller intervals
         batch_bg = intervals_to_bg(batch_intervals)
-    return batch_bg
+        df_to_bedGraph(batch_bg, outfile)
 
 
-def writer(args, res_queue, input_file_path):
+def writer(args, res_queue, prefix):
```

Comment:
can you add doctoring here explaining the arguments? I know this function didn't have it from the beginning, but probably a good time to start now :)

Reply:
done